core/http/client/src/main/java/org/eclipse/rdf4j/http/client/RemoteValidation.java

```diff
@@ -0,0 +1,48 @@
+package org.eclipse.rdf4j.http.client;
+
+import org.eclipse.rdf4j.model.Model;
+import org.eclipse.rdf4j.model.Resource;
+import org.eclipse.rdf4j.rio.RDFFormat;
+import org.eclipse.rdf4j.rio.Rio;
+
+import java.io.IOException;
+import java.io.StringReader;
+
+public class RemoteValidation implements ModelInterface {
```

Comment:
Intention is to support streaming validation reports down the line. Hopefully I can build that as part of this branch, so that I can get the requirements down from the get go.

Reply:
Can we make it package private? I'm a little wary of introducing public interfaces and classes for this.